content/setup/overview.md

```diff
@@ -7,6 +7,19 @@ menu = "installation"
 toc = true
 +++
 
+# Features support
+
+| Feature/Remote            | GitHub  | GitLab  | BitBucket           | Gogs    |
+|---------------------------|---------|---------|---------------------|---------|
+| Supported version         | --/--   | 8.2+    | --/--               | --/--   |
+| VCS                       | **git** | **git** | **git ( only )**    | **git** |
```

Comment:
we have a plugin for mercurial that should work with Bitbucket mercurial repositories, however, this isn't something I've thoroughly tested:

```
clone:
  image: hg
```